Geometry/GEMGeometryBuilder/python/gemGeometry_cff.py

```diff
@@ -2,4 +2,8 @@
 
 from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
 
+from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+
 dd4hep.toModify(gemGeometry, fromDDD = False, fromDD4hep = True)
+
+run3_GEM.toModify(gemGeometry, applyAlignment = True)
```

Comment:
naively this modification (and the dd4hep line above, but thats not new) should instead be in gemGeometry_cfi.py to keep the module definition in one place.

Reply:
An established pattern for such cases would be to have the ESModule to generate `gemGeometryDefault_cfi.py` and then to have a hand-written `gemGeometry_cfi.py` along
```py
from Geometry.GEMGeometryBuilder.gemGeometryDefault_cfi import gemGeometryDefault as _gemGeometryDefault

gemGeometry = _gemGeometryDefault.clone()

dd4hep.toModify(gemGeometry, fromDDD = False, fromDD4hep = True)
run3_GEM.toModify(gemGeometry, applyAlignment = True)
```
(such a reorganization may be outside of the scope of this PR)